### PR TITLE
Site translation

### PR DESCRIPTION
**Pull request by gsouf**
# Translations

Firstly i have created a directory `src` at the root that contains the untranslated views, and the trnaslation strings.

The translation is handled by glup : http://git.io/uML7fQ

It will compile the the views from `src/views` into `public/lang`. The file `public/index.html` is a copy of `public/lang/en/index.html`
# Language Detection

The language is detected from the browser, if the user comes for the first time we redirect it to the good language. The next times he comes we dont redirect it.
# Link

Right now the links for each langs are `domain.com/langs/[lang]/index.html`, but because of angular it doesnt work nice right now. I assume that we will change these urls to something like  `[lang].domain.com/index.html`
# Add a language

To add a language : 
- create the file `src/locales/[lang].yml` 
- add the language to the gulpfile (http://git.io/tKIYzQ)
- add the language to the list of client languages (http://git.io/o616YA)
- Run gulp

### CODE REVIEW

**Comment by ovr** [comment]
@SneakyBobito 
ping
jQuery and jQuery.cookie Is not removed
https://github.com/zephir-lang/website/pull/1/files#diff-b9e12334e9eafd8341a6107dd98510c9R34

**Comment by gsouf** [comment]
Pong - Done !

File: public/css/my-foundation.css [hunk 1]
```diff
@@ -1172,11 +1172,11 @@ button, .button {
   padding-left: 2rem;
```
Comment (ovr) [inline public/css/my-foundation.css:1172]: Why this file is changed?
It's auto generated https://github.com/zephir-lang/website/blob/master/gulpfile.js#L8

Comment (gsouf) [inline public/css/my-foundation.css:1172]: I didnt change it

**Comment by ovr** [comment]
Thanks

**Pull request merged**